### PR TITLE
Add js debugger using VsCode built-in Node debugger

### PR DESCRIPTION
**Pull request by kiennq**
Issue #51

### CODE REVIEW

**Comment by yyoncho** [comment]
Thank you for your work on `dap-mode`!

When I was testing the adapter I saw that sometimes the adapter returns uri prefixed with `<node_internal>/file:` . I was debugging a simple file out.js with no project file or whatever. Have you seen something like that?

File: dap-node.el [hunk 1]
```diff
@@ -0,0 +1,67 @@
+;;; dap-node.el --- Debug Adapter Protocol mode for Node      -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2019  Ivan Yonchovski
+
+;; Author: Ivan Yonchovski <yyoncho@gmail.com>
```
Comment (yyoncho) [inline dap-node.el]: Change the author to you.

File: dap-node.el [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;; URL: https://github.com/yyoncho/dap-mode
+;; Package-Requires: ((emacs "25.1") (dash "2.14.1") (lsp-mode "4.0"))
+;; Version: 0.2
+
+;;; Commentary:
+;; Adapter for https://github.com/nodeide/vscode-node
```
Comment (yyoncho) [inline dap-node.el]: this url is 404.

**Comment by yyoncho** [comment]
Ok. So I tested the file that used to fail and now it works fine. So, I will merge it after you fix the doc issues.

File: dap-node.el (hunk 1 above)
Comment (kiennq) [reply dap-node.el]: Fixed

File: dap-node.el (hunk 2 above)
Comment (kiennq) [reply dap-node.el]: Fixed

**Comment by kiennq** [comment]
I've updated the doc, can you please check?

**Pull request merged**